frontend/src/templates/Field/Table/ColumnCell.tsx

```diff
@@ -138,7 +140,8 @@ export const ColumnCell = ({
       isInvalid={!!get(errors, inputName)}
     >
       <FormLabel
-        as={isMobile ? undefined : VisuallyHidden}
+        // display column header in print and mobile modes
+        as={isPrint ? undefined : isMobile ? undefined : VisuallyHidden}
```

Comment:
Heeeeeyyy?? github dropped my second comment 😭 Am not sure what happened but I'm sorry about it. I had this extra not:
```suggestion
        as={isPrint || isMobile ? undefined : VisuallyHidden}
```

Reply:
I pushed the change to save a bit of time 😅

Reply:
sorry!!! was afk